develop/blockchains/deployment/generate-chain-specs.md

```diff
@@ -1,4 +1,136 @@
+
+- Initial account balances
+- Accounts that are initially part of a governance council
+- The account that controls the `sudo` key
+- Any other genesis state for a pallet
+
+Nodes also require the compiled Wasm to execute the runtime logic on the chain, so the initial runtime must also be supplied in the chain specification. For a more detailed look at customizing the genesis chain specification, be sure to check out the [Polkadot SDK Docs](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/chain_spec_genesis/index.html){target=_blank}.
+
+## Declaring storage items for a runtime
+
+A runtime usually requires some storage items to be configured at genesis. This includes the initial state for pallets, for example, how much balance specific accounts have, or which account will have sudo permissions.
+
+These storage values are configured in the genesis portion of the chain specification. You can create a [patch](https://paritytech.github.io/polkadot-sdk/master/sc_chain_spec/index.html#chain-spec-formats){target=_blank} file and ingest it using the [`chain-spec-builder`](https://paritytech.github.io/polkadot-sdk/master/staging_chain_spec_builder/index.html){target=_blank} utility, that is explained in the [Creating a Custom Chain Specification](#creating-a-custom-chain-specification) section.
+
+## Chain specification JSON format
+
+Users generally work with the JSON format of the chain specification. Internally, the chain specification is embedded in the [`GenericChainSpec`](https://paritytech.github.io/polkadot-sdk/master/sc_chain_spec/struct.GenericChainSpec.html){target=\_blank} struct, with specific properties accessible through the [`ChainSpec`](https://paritytech.github.io/polkadot-sdk/master/sc_chain_spec/trait.ChainSpec.html){target=\_blank} struct. The chain specification includes the following keys:
+
+- `name` - the human-readable name for the network
+- `id` - the machine-readable identifier for the network
+- `chainType` - the type of chain to start (refer to [ChainType](https://paritytech.github.io/polkadot-sdk/master/sc_chain_spec/enum.ChainType.html){target=\_blank} for more details)
+- `bootNodes` - a list of multiaddresses belonging to the chain's boot nodes
+- `telemetryEndpoints` - an optional list of multiaddresses for telemetry endpoints, with verbosity levels ranging from 0 to 9 (0 being the lowest verbosity)
+- `protocolId` - the optional protocol identifier for the network
+- `forkId` - an optional fork ID that should typically be left empty; it can be used to signal a fork at the network level when two chains share the same genesis hash
+- `properties` - custom properties provided as a key-value JSON object
+- `codeSubstitutes` - an optional mapping of block numbers to Wasm code
+- `genesis` - the genesis configuration for the chain
```

Comment:
```suggestion
- **`name`** - the human-readable name for the network
- **`id`** - the machine-readable identifier for the network
- **`chainType`** - the type of chain to start (refer to [`ChainType`](https://paritytech.github.io/polkadot-sdk/master/sc_chain_spec/enum.ChainType.html){target=\_blank} for more details)
- **`bootNodes`** - a list of multiaddresses belonging to the chain's boot nodes
- **`telemetryEndpoints`** - an optional list of multiaddresses for telemetry endpoints with verbosity levels ranging from 0 to 9 (0 being the lowest verbosity)
- **`protocolId`** - the optional protocol identifier for the network
- **`forkId`** - an optional fork ID that should typically be left empty; it can be used to signal a fork at the network level when two chains share the same genesis hash
- **`properties`** - custom properties provided as a key-value JSON object
- **`codeSubstitutes`** - an optional mapping of block numbers to Wasm code
- **`genesis`** - the genesis configuration for the chain
```